nuxeo-core/nuxeo-core-mimetype/src/test/java/org/nuxeo/ecm/platform/mimetype/TestMimetypeRegistryService.java

```diff
@@ -254,4 +256,46 @@ public void testGetMimetypeEntryByMimetype() {
         assertNotNull(entry);
         assertEquals("application/msword", entry.getNormalized());
     }
+
+    @Test
+    public void iCanRetrieveNormalisedMimetype() {
+        verifyNormalisedMimetype("application/zip", List.of("application/zip", "application/x-zip-compressed"));
+        verifyNormalisedMimetype("text/plain", List.of("text/plain"));
+        verifyNormalisedMimetype("text/x-rst", List.of("text/x-rst", "text/restructured"));
+        verifyNormalisedMimetype("text/python-source", List.of("text/python-source", "text/x-python"));
+        verifyNormalisedMimetype("application/vnd.ms-excel", List.of("application/vnd.ms-excel"));
+        verifyNormalisedMimetype("application/pdf", List.of("application/pdf"));
+        verifyNormalisedMimetype("application/vnd.sun.xml.writer.template",
+                List.of("application/vnd.sun.xml.writer.template"));
+
+        verifyNormalisedMimetype("application/photoshop",
+                List.of("application/photoshop", "application/x-photoshop", "image/photoshop", "image/x-photoshop",
+                        "image/psd", "image/x-psd", "image/vnd.adobe.photoshop", "application/psd", "zz-application",
+                        "zz-winassoc-psd"));
```

Comment:
I wonder if it's necessary to test all these normalized mime types in a unit test (we don't want to validate the whole contributions but just the API)?
Also, it could be a pain to maintain, especially for `application/photoshop` for instance as we could be adding some new ones often.

Reply:
yes, the idea here is not to test the whole contribution, i just take more example/use cases to test. if we add a new `application/photoshop` we are not obligated to add it